shared-helpers/src/locales/es.json

```diff
@@ -406,7 +406,7 @@
   "counties.Solano": "Solano",
   "counties.Sonoma": "Sonoma",
   "counties.fullname.Alameda": "Condado de Alameda",
-  "counties.fullname.Contra Costa": "Condado de Contra Costa",
+  "counties.fullname.ContraCosta": "Condado de Contra Costa",
```

Comment:
Is removing the space a typo? https://github.com/metrotranscom/doorway/pull/225/files#diff-2adb536fd49c9a9de3b0305b9a8a4949a377e68e9ae1d6558c31c49f6068bb05R185 has a space.

Reply:
This is intended. The message name in general.json is "counties.fullname.ContraCosta" so we have to use the same for all other languages. The name was not being properly translated, this is necessary.

Reply:
So odd! I swore I already fixed this. Maybe I haven't pushed that PR yet. Great find @ifranch!!!